src/test/ui/bound-suggestions.rs

```diff
@@ -40,4 +40,9 @@ fn test_many_bounds_where<X>(x: X) where X: Sized, X: Sized {
     //~^ ERROR doesn't implement
 }
 
-pub fn main() { }
+trait Foo<T> {
```

Comment:
Please also add tests for the other cases (already existing where bounds and already existing super traits). I guess technically even one with both already existing :laughing:

Reply:
Done.